### PR TITLE
refactor: Added separate file for Api handlers and api url

### PR DESCRIPTION
**Pull request by errorr404**
Added separate file for Api handlers and api url

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  });
+const sendPut = (url, data = null) =>
+  axios.put(url, data, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+
+// the get request who uses access token as header goes under sendGetPrivate
+const galleryVoatingApi = (galleryHash, vote) =>
```
Comment (Arjun-sna) [inline src/api.js]: `galleryVoatingApi` typo?

File: src/apiUrl.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export const userDetailsUrl = username => `https://api.imgur.com/3/account/${username}`;
```
Comment (Arjun-sna) [inline src/apiUrl.js]: `https://api.imgur.com/3` put this in a const variable and reuse

File: src/api.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  sendGetPrivate(galleryVoatingUrl(galleryHash, vote));
+
+const favoriteAlbumApi = albumId => sendPost(favoriteUrl(albumId));
+
+const reportUserApi = (commentId, reason) =>
```
Comment (Arjun-sna) [inline src/api.js]: we can remove `Api` in all function names. The function names should be a verb in general.

File: src/api.js (hunk 1 above)
Comment (errorr404) [reply src/api.js]: https://github.com/Codebrahma/imgur-client-app/pull/15/commits/aff1b1ef88a22d83dfeb6f80c9065279f8fcc96f
Fixed.

File: src/apiUrl.js (hunk 2 above)
Comment (errorr404) [reply src/apiUrl.js]: https://github.com/Codebrahma/imgur-client-app/pull/15/commits/fe0b7057b810bc1350ea639e58cd4856f00dca39
Done !

File: src/api.js (hunk 3 above)
Comment (errorr404) [reply src/api.js]: https://github.com/Codebrahma/imgur-client-app/pull/15/commits/d98d2637c0efa7e5cfa622a37ef371cd20fcbb44
Done !

**Pull request merged**